### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,12 +11,12 @@
...
-        "silverstripe/assets": "^1.3"
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "silverstripe/vendor-plugin": "^2",
+        "dnadesign/silverstripe-elemental": "^5",
+        "silverstripe/cms": "^5",
+        "silverstripe/assets": "^2"
     },
     "require-dev": {
         "silverstripe/recipe-testing": "^2"
```
Comment (emteknetnz) [inline composer.json]:
```suggestion
        "silverstripe/recipe-testing": "^3"
```

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (sabina-talipova) [reply composer.json]: DONE

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Unit test failures

**Comment by sabina-talipova** [comment]
Reason for failed test is that composer installed dnadesign/silverstripe-elemental 5.0.0-alpha1.
https://github.com/silverstripe/silverstripe-elemental-fileblock/actions/runs/3753124017/jobs/6612598079#step:9:432
Passed tests in local.

**Comment by maxime-rainville** [comment]
I've created a alpha2 release for elemental to see if this fixes it.

**Review by maxime-rainville — APPROVED** [review]
The prefer-lowest build is still failing because it still wants to install alpha1.

I'm going to call this a problem with the CI set up.

**Pull request merged**